tests/integration/springer/test_dag_process_file.py

```diff
@@ -47,39 +52,56 @@ def extract_zip_to_article(zip_filename):
     return article
 
 
-def test_dag_loaded(dag: DAG):
-    assert dag is not None
-    assert len(dag.tasks) == 4
-
-
-def test_dag_run(dag: DAG, article):
-    id = datetime.datetime.utcnow().strftime(
-        "test_springer_dag_process_file_%Y-%m-%dT%H:%M:%S.%f"
-    )
-    dagrun = dag.create_dagrun(
-        DagRunState.QUEUED,
-        run_id=id,
-        conf={"file": base64.b64encode(ET.tostring(article)).decode()},
-    )
-    wait().at_most(60, SECOND).until(lambda: __test_dagrun_state(dagrun))
-
-
-def test_dag_run_no_input_file(dag: DAG):
-    id = datetime.datetime.utcnow().strftime(
-        "test_springer_dag_process_file_%Y-%m-%dT%H:%M:%S.%f"
-    )
-    dagrun = dag.create_dagrun(DagRunState.QUEUED, run_id=id)
-    wait().at_most(60, SECOND).until(lambda: __test_dagrun_fails(dagrun))
-
-
-def test_dag_parse_file(article):
-    springer_parse_file(
-        params={"file": base64.b64encode(ET.tostring(article)).decode()}
-    )
+class TestClassSpringerFilesHarvesting:
+    def test_dag_loaded(self, dag: DAG):
+        assert dag is not None
+        assert len(dag.tasks) == 4
+
+    def test_dag_run(self, dag: DAG, dag_was_paused: bool, article: ET):
+        id = datetime.datetime.utcnow().strftime(
```

Comment:
better not to call the vars in Python `id`, check https://stackoverflow.com/questions/77552/id-is-a-bad-variable-name-in-python